account_statement_import/views/account_journal.xml

```diff
@@ -11,6 +11,9 @@
         <field name="model">account.journal</field>
         <field name="inherit_id" ref="account.account_journal_dashboard_kanban_view" />
         <field name="arch" type="xml">
+            <kanban position="attributes">
+                <attribute name="create">1</attribute>
```

Comment:
This one is weird, as creating a journal from the dashboard is something outside the module scope.

Reply:
The order is used to show the buttons on top. Otherwise it is not rendered and we cannot show the import button